lms/djangoapps/django_comment_client/management/commands/rename_user.py

```diff
@@ -0,0 +1,22 @@
+"""
+Change the username of an existing user
+"""
+
+from django.core.management.base import BaseCommand, CommandError
+from .. import management_utils
+
+
+class Command(BaseCommand):
+    """
+    invoke this manage.py command from the console as follows:
+    python manage.py lms rename_user <username1> <username2> --settings=devstack
+    """
+    args = '<old_username> <new_username>'
+    help = 'Modify the username of an existing user'
+
+    def handle(self, *args, **options):
+        if len(args) != 2:
+            raise CommandError('Usage is rename_user {0}'.format(self.args))
```

Comment:
To avoid hard-coding the command name, you could try the following:

```python
from os.path import basename
# ...
if len(args) != 2:
    command_name = '.'.join(basename(__file__).split('.')[:-1])
    raise CommandError(
        "Usage is {command_name} {command_args}'.format(
            command_name=command_name,
            command_args=self.args,
         )
    )
```